src/components/EmojiPicker/EmojiSkinToneList.js

```diff
@@ -59,17 +59,18 @@ class EmojiSkinToneList extends Component {
                                 () => this.setState(prev => ({isSkinToneListVisible: !prev.isSkinToneListVisible}))
                             }
                             style={[
-                                styles.pv1,
```

Comment:
Let's add this back to the emoji but include this on the emoji height so that we can cover this value. Motive should always be to keep the same UI. I see that UI is little bit different from staging.

Reply:
This is not the reason for slight change in UI. It is due to the introduction of `EMOJI_PICKER_ITEM_HEIGHT` and `variables.emojiLineHeight`.